Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java

```diff
@@ -1,43 +1,135 @@
 package competition.subsystems.agitator;
 
-import javax.inject.Singleton;
-
 import org.apache.log4j.Logger;
-
+import com.ctre.CANTalon.FeedbackDevice;
 import com.ctre.CANTalon.TalonControlMode;
-import com.google.inject.Inject;
-
+import competition.subsystems.RobotSide;
+import edu.wpi.first.wpilibj.Timer;
 import xbot.common.command.BaseSubsystem;
 import xbot.common.controls.actuators.XCANTalon;
 import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.math.PIDPropertyManager;
+import xbot.common.properties.BooleanProperty;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
-@Singleton
-public class AgitatorSubsystem extends BaseSubsystem{
+
+public class AgitatorSubsystem extends BaseSubsystem {
```

Comment:
If this now is responsible for only a single side, it should be called `AgitatorSideSubsystem` or `EggbeaterSubsystem` -- the whole system is the agitator, the individual arms are "eggbeaters".